lib/Vault.js

```diff
@@ -42,6 +42,9 @@ class Vault {
      * @return {undefined}
     */
     authenticateV4Request(params, requestContexts, callback) {
+        if (process.env.NO_AUTH) {
```

Comment:
I would suggest setting up auth like S3 does, using an `authData.json` and we can use an env. var. like
`UTAPI_AUTH=MEMORY`.